<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormVehiculoRubro</class>
 <widget class="QDialog" name="FormVehiculoRubro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>281</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Icono</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Marca</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LabelImage">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Imagen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>303</width>
     <height>236</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QTabWidget::pane { 
font: bold 14px;
   background: transparent; 
  margin: 0px,0px,0px,0px;
   border: 0px solid transparent;  
  }

QTabBar::tab {
    background: transparent;
    border: 2px solid transparent;
    border-bottom-color: transparent; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}

</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::East</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="../Recursos.qrc">
      <normaloff>:/Imagenes/iconos/add.png</normaloff>:/Imagenes/iconos/add.png</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Imagen</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Codigo:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="2">
        <widget class="QToolButton" name="ButtonArchivoImagen">
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="LineNombre">
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhFormattedNumbersOnly</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="LineImagen">
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="LineCodigo">
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="ButtonGuardar">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Recursos.qrc">
           <normaloff>:/Imagenes/iconos/guardar.png</normaloff>:/Imagenes/iconos/guardar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonModificar">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Recursos.qrc">
           <normaloff>:/Imagenes/iconos/editar.png</normaloff>:/Imagenes/iconos/editar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonEliminar">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Recursos.qrc">
           <normaloff>:/Imagenes/iconos/delete.png</normaloff>:/Imagenes/iconos/delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonRegresar">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Recursos.qrc">
           <normaloff>:/Imagenes/iconos/izquierda.png</normaloff>:/Imagenes/iconos/izquierda.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
